releases/v1.20/vsphere-cloud-controller-manager.yaml

```diff
@@ -0,0 +1,253 @@
+---
```

Comment:
Can we split the manifest update into a separate PR so the panic is easier to cherry-pick to older release?

Reply:
gotcha, created another PR: https://github.com/kubernetes/cloud-provider-vsphere/pull/487/files for adding manifests